components/Map/index.js

```diff
@@ -0,0 +1,85 @@
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
+import styled from "styled-components";
+import * as L from "leaflet";
+import "leaflet/dist/leaflet.css";
+import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
+import "leaflet-defaulticon-compatibility";
+import CurrentOutivityMarker from "./CurrentOutivityMarker";
+import Link from "next/link";
+import useLocalStorage from "use-local-storage";
+
+export default function Map({ currentOutivity = {}, outivities }) {
+  const [userPosition, setUserPosition] = useLocalStorage("userPosition", {
+    lat: "",
+    lng: "",
+  });
+
```

Comment:
Damit wir die userPosition in der CurrentOutivityMarker.js setzen können, müssen wir hier eine handler-function mitgeben: 
```suggestion
function handleSetUserPosition(userCoordinates) {
    setUserPosition(userCoordinates);
  }
```